src/routes/admins.js

```diff
@@ -1,3 +1,100 @@
-export default(apiServer) => {
-    
+import { list, readOne, updateOne, deleteOne } from 'mongoose-crudl'
+import jwt from 'jsonwebtoken';
+import AdminModel from '../models/Admin.js'
+import MethodNotAllowedError from '../errors/MethodNotAllowedError.js';
+import allowAccessTo from '../helpers/allowAccessTo.js'
+import crypto from 'crypto';
+
+export default (apiServer) => {
+  const secrets =  process.env.SECRETS.split(' ');
+
+  apiServer.get('/v1/admins/', async req => {
+    try {
```

Comment:
You don't actually need to wrap these in try-catch blocks, because there is a big try-catch block in the request handler in express-async-api https://github.com/gyulanemeth/express-async-api/blob/master/src/index.js#L6

Just return from this function what should be sent back to the client, and throw an error if something goes wrong.

Right now, there is a global error handler callback, we will need to generate the proper responses there. (a few lines below the line I sent above)